content/themes/semantic-ui/amazon/index.slim

```diff
@@ -9,8 +9,9 @@
       a.item data-tab="views" Views
       a.item data-tab="modules" Modules
 
-.ui.demo.container
+.ui.demo.container.basic.segment
```

Comment:
It's OK, I think maybe you should keep this question in your head and try to figure it out in your spare time.